src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/custom.py

```diff
@@ -51,3 +51,24 @@ def list_vm(client, resource_group, lab_name, order_by=None, top=None,
 
     return client.list(resource_group, lab_name,
                        expand=expand, filter=filters, top=top, order_by=order_by)
+
+
+def get_formula(client, resource_group, lab_name, name, export_artifacts=None):
+    """ Command to show formula from the Azure DevTest Lab """
+    formula = client.get_resource(resource_group_name=resource_group, lab_name=lab_name, name=name)
```

Comment:
Alternatively, consider leaving the "show" operation as is and having "export-artifacts" as a convenience command. I think that would be more consistent with existing commands like this (i.e. KeyVault's get-default-policy).

Also, you could implement this by simply reflecting on the get operation and using a transformer method to massage the result (since that is really all _export_artifacts is doing). This would avoid the potential pitfall that @troydai mentions as well since it would be clear that this is an output transform only.